8.1_fib.js

```diff
@@ -0,0 +1,15 @@
+var fib = function (n) {
+   if(n === 1){
+      return 1;
+   }
+   if(n === 2){
```

Comment:
consider making these else if statements so that later conditionals can be short circuited